src/supermarket/app/helpers/cookbook_versions_helper.rb

```diff
@@ -66,4 +66,29 @@ def render_document(content, extension)
       content
     end
   end
+
+  def chef_versions_string(chef_versions)
+    versions_string(chef_versions)
+  end
+
+  def ohai_versions_string(ohai_versions)
```

Comment:
Yey, helpers! I wonder, though, if `versions_string()` could be promoted to public and used directly. There is currently no difference in rendering a collection of version constraint ranges.